components/movies/MovieDetails.bs

```diff
@@ -591,8 +599,6 @@ sub SetDefaultAudioTrack(itemData)
             end if
         end if
     end for
```

Comment:
What if there's no user selection, not configured to play default, and no audio tracks matching preferred language?

Previously we would return the 1st found audio track, but with these extra conditions it's possible m.top.selectedAudioStreamIndex doesn't get set with new values.

I think we need a fallback so if all the conditions fail, we set it to the 1st found track like we used to.

Reply:
I thought I went back and added that but must have missed it.  Thanks 👍